rules-presets/ui-modifications/mark-long-living-sessions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Mark Long Living Sessions
```

Comment:
Google tends to present results for "long-lived sessions" rather than "long living sessions". Apply at your discretion, I will not be marking any more instances of the term.